example/ts/src/exampleApiServer.ts

```diff
@@ -4,7 +4,7 @@
 import * as bodyParser from 'body-parser';
 import * as express from 'express';
 import { IServiceResult } from 'facility-core';
-import { IExampleApi, IGetWidgetsRequest, IGetWidgetsResponse, ICreateWidgetRequest, ICreateWidgetResponse, IGetWidgetRequest, IGetWidgetResponse, IDeleteWidgetRequest, IDeleteWidgetResponse, IEditWidgetRequest, IEditWidgetResponse, IGetWidgetBatchRequest, IGetWidgetBatchResponse, IGetWidgetWeightRequest, IGetWidgetWeightResponse, IGetPreferenceRequest, IGetPreferenceResponse, ISetPreferenceRequest, ISetPreferenceResponse, IGetInfoRequest, IGetInfoResponse, INotRestfulRequest, INotRestfulResponse, IKitchenRequest, IKitchenResponse, IWidget, IWidgetJob, IPreference, IObsoleteData, IKitchenSink, WidgetField, ObsoleteEnum } from './exampleApiTypes';
+import { IExampleApi, IGetWidgetsRequest, IGetWidgetsResponse, ICreateWidgetRequest, ICreateWidgetResponse, IGetWidgetRequest, IGetWidgetResponse, IDeleteWidgetRequest, IDeleteWidgetResponse, IEditWidgetRequest, IEditWidgetResponse, IGetWidgetBatchRequest, IGetWidgetBatchResponse, IGetWidgetWeightRequest, IGetWidgetWeightResponse, IGetPreferenceRequest, IGetPreferenceResponse, ISetPreferenceRequest, ISetPreferenceResponse, IGetInfoRequest, IGetInfoResponse, INotRestfulRequest, INotRestfulResponse, IKitchenRequest, IKitchenResponse, IWidget, IWidgetJob, IPreference, IObsoleteData, IKitchenSink, WidgetField, ObsoleteEnum, ExampleApiErrors } from './exampleApiTypes';
```

Comment:
If `ExampleApiErrors` is imported, should it be used somewhere in this file? If it should be used, how is this file generated? Same question applies to `conformance/ts/src/conformanceApi.ts`

Reply:
I forgot that we generate server code. I wonder if anyone uses it... Strictly speaking, the errors should probably be added to `standardErrorCodes`. The HTTP error codes are specified in `HttpServiceInfo.ErrorSets`.

Reply:
`conformanceApi.ts` is client code, not server code.

Reply:
> server code. I wonder if anyone uses it...

The only projects I'm aware of that used it are no longer maintained.

Reply:
> `conformanceApi.ts` is client code, not server code.

@ejball I'm not sure what changes you're requesting for this? The conformance FSD file [has](https://github.com/FacilityApi/FacilityJavaScript/blob/9c67c70165f7990af11b69681c01a45100bc0370/conformance/ConformanceApi.fsd#L373-L382) an error set so [this change](https://github.com/FacilityApi/FacilityJavaScript/pull/57/commits/6a08084e37f0571abb9b85f95f73066774f8aafd) means that the error set gets added to the generated types file in `conformanceApiTypes.ts`.

Is your request that `ApiError` no longer be imported into `conformanceApi.ts` (and `exampleApi.ts`) ? If so, I'm not sure how to make that change

Reply:
Oops you just re-approved 😄

Reply:
> conformanceApi.ts is client code, not server code.

> ... I'm not sure what changes you're requesting...

It's likely you weren't requesting changes, you were just commenting that since the code is for the client (not server) there's no need to add to the error set codes to any standard error codes. I interpreted the comment as part of the request for changes.